tests/app/shared/issue/title/title.component.spec.ts

```diff
@@ -12,14 +12,16 @@ describe('TitleComponent', () => {
   let thisIssue: Issue;
   let formBuilder: any;
   let phaseService: PhaseService;
+  let dialogService: any;
```

Comment:
Type here should be `DialogService`

Reply:
The `dialogService` mock is created using `jasmine.createSpyObj` with return type `any`, so I've kept the type as `any` for consistency.

Reply:
hmm we should not use `any` as much as possible since it takes away the benefit of typescript.
try `jasmine.SpyObj<IMyService> = jasmine.createSpyObj<IMyService>('service',['method']);`